controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -51,7 +51,7 @@
 import com.yahoo.vespa.hosted.controller.deployment.DeploymentSteps;
 import com.yahoo.vespa.hosted.controller.deployment.DeploymentTrigger;
 import com.yahoo.vespa.hosted.controller.permits.ApplicationPermit;
-import com.yahoo.vespa.hosted.controller.permits.PermitStore;
+import com.yahoo.vespa.hosted.controller.permits.AccessControl;
```

Comment:
Rename package as well? `security`?

Reply:
Perhaps. I want to rename it. Also, I'm changing Permit to Claim. This is the right word.

Reply:
But I'll keep the signatures as they are. Trying to change it (again) yesterday, I realised there were more things going into them, and trying to model it more accurately than Claim is just pointless, and doesn't accommodate new types either.